modules/glnexus/main.nf

```diff
@@ -0,0 +1,48 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process GLNEXUS {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::glnexus=1.4.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "docker://clinicalgenomics/glnexus:v1.4.1"
+    } else {
+        container "clinicalgenomics/glnexus:v1.4.1"
+    }
+
+    input:
+    tuple val(meta), path(gvcfs)
+
+    output:
+    tuple val(meta), path("*.bcf"), emit: bcf
+    path "*.version.txt"          , emit: version
+
+    script:
+    def software = getSoftwareName(task.process)
+    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+
+    // Make list of GVCFs to merge
+    def input = ""
+    for (gvcf in gvcfs) {
+        input += " ${gvcf}"
+    }
```

Comment:
This logic should work rather than looping over the files 

https://github.com/nf-core/modules/blob/77a2895785587219ee2b9d0ea16fa0d71ad14f3a/modules/cat/cat/main.nf#L29